vendor/bazel_json/lib/BUILD

```diff
@@ -1,4 +1,5 @@
 package(default_visibility = ["//:__pkg__"])
+
```

Comment:
After double-checking, the modifications to `bazel_json` were not due to an update but to buildifier. And they seem unnecessary since CI still passes after removing this commit (so I am  merging without them).

About replacing `bazel_json` with bazel json module. Looking at the documentation of https://bazel.build/rules/lib/json, I do not see a way to test if a json string is valid before trying to parse it, in order to customize the error message. And we use the `fail_on_invalid = False` functionnality of `bazel_json` for this once ([here](https://github.com/tweag/rules_haskell/blob/master/haskell/cabal.bzl#L1700-L1705)).

Reply:
Thanks for looking into this! Makes sense!

> I do not see a way to test if a json string is valid before trying to parse it, in order to customize the error message. And we use the `fail_on_invalid = False` functionnality of `bazel_json` for this once ([here](https://github.com/tweag/rules_haskell/blob/master/haskell/cabal.bzl#L1700-L1705)).

I see. Well, given that that code fails immediately itself after observing an error, it doesn't look like a big loss. So, I think we could still remove `bazel_json` and replace it with the builtin JSON parser.